README.md

```diff
@@ -43,22 +43,29 @@ The Docker environment is recommended because of the fewer setup steps required
  * Install [git](https://git-scm.com/downloads)
  * Install [Git-LFS](https://git-lfs.github.com/)
 
+### Forking the repo 
+
+[Fork](https://github.com/odk-x/docs/fork) the odk-x/docs repository, so that a copy of odk-x/docs gets created in your github account.
+
 ### Cloning the repo
 
-Clone the docs repo into a directory you want the ODK-X docs files to be located. For example, navigate to the directory you want the files to be located using the "cd" (Change Directory) command on the command line:.
+Firstly, change to the directory(if there's one) you want to clone the ODK-X docs repository on your local machine.
 ```
 cd <DIRECTORY>
 ```
 
-Then use git to get a copy of the ODK-X documentation files.
+Then use git to get a copy of the ODK-X docs repository on your local machine.
 
 ```
-git clone https://github.com/odk-x/docs.git
+git clone <forked_repo_link>
```

Comment:
Why did this change to <forked_repo_link>? Is the old link not correct? If it was correct, recommend retaining it, <forked_repo_link> is ambiguous.

Reply:
and also it's a good practice for contributor to fork the repo you need to work on and make changes to it